SampleObjC/sdk-ios-objc/OCViewController.m

```diff
@@ -0,0 +1,109 @@
+//
+//  ViewController.m
+//  sdk-ios-objc
+//
+//  Created by Simon Hamadene on 16/10/2018.
+//  Copyright © 2018 Yoti Limited. All rights reserved.
+//
+
+#import "OCViewController.h"
+#import <YotiButtonSDK/YotiButtonSDK.h>
+#import "OCProfileViewController.h"
+
+@interface OCViewController () <YTBSDKDelegate, YTBBackendDelegate>
+@property (weak, nonatomic) IBOutlet YotiButton *rememberMe;
+@property (weak, nonatomic) IBOutlet YotiButton *selfieAuth;
```

Comment:
Please make them `nonnull`

Reply:
They are `weak` they can't be `nonnull`.